test/test_viewing_area.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
Put my comment praising the test in the wrong spot. 

This is great. Hopefully we can keep adding tests of the individual class functions. Exactly what @kjmerf has been talking about.

Reply:
Agree, well done here. My only question is: what's the advantage to using the unit test module?

Reply:
@kjmerf good question. I had not used pytest as a test runner and I was actually pleasantly surprised to see that recognizes ordinary asserts. I've always used python -m unittest discover as my test runner; now that we're using pytest this is a fair question. Here are the things I know about:

- Inherited methods setUp and tearDown, while not used in my examples, can be useful if you need to spin up something and tear it down before every test.
- unittest.main() can replace main and give you a nice way to run each unit test as a module, e.g., `python -m unittest test.test_mything`

I also thought of unittest as a standard, but I have to be honest that I rarely use the two features I brought up in the list above.

Reply:
Thanks. I think unit test is standard but I’ve never really understood why.